.github/workflows/pack-android-sdks.yml

```diff
@@ -0,0 +1,93 @@
+name: Package Android SDKs
```

Comment:
It's annoying to run win/linux just to package SDK for all the platforms.
I added this action we can run manually and prepare SDKs specified in `sdk.py`